.travis.yml

```diff
@@ -12,22 +13,25 @@ env:
   - DJANGO=1.10
   - DJANGO=1.11
   - DJANGO=2.0
+  - DJANGO=2.1
 matrix:
   exclude:
     # Python/Django combinations that aren't officially supported
     - { python: 3.5, env: DJANGO=1.7 }
     - { python: 3.6, env: DJANGO=1.7 }
     - { python: 3.6, env: DJANGO=1.8 }
-    - { python: 3.3, env: DJANGO=1.9 }
     - { python: 3.6, env: DJANGO=1.9 }
-    - { python: 3.3, env: DJANGO=1.10 }
     - { python: 3.6, env: DJANGO=1.10 }
-    - { python: 3.3, env: DJANGO=1.11 }
     - { python: 2.7, env: DJANGO=2.0 }
-    - { python: 3.3, env: DJANGO=2.0 }
+    - { python: 2.7, env: DJANGO=2.1 }
+    - { python: 3.4, env: DJANGO=2.1 }
```

Comment:
**Note to nit-pickers:** Strictly speaking, Django 2.0+ doesn't support Python 2.7, and Django 2.1+ doesn't even support Python 3.4 (you're right), but we keep it in there because we want to be as nice as Guido.  :smile: